doc/password-hashing-functions/index.md

```diff
@@ -0,0 +1,97 @@
+# Password hashing fucntions
+
+!!! info
+
+    **Status**: WIP RF
+    
+    **Level**: ?
```

Comment:
Suggest this is [Level 3](https://gig-cymru-nhs-wales.github.io/Architecture-Decision-Records/design-authority/dhcw/architecture-decision-record-process/#level-3-organisation-wide-decisions)